MVVM Module/Source/Feed Presentation/FeedViewModel.swift

```diff
@@ -20,15 +20,27 @@ final class FeedViewModel {
 	
 	var onLoadingStateChange: Observer<Bool>?
 	var onFeedLoad: Observer<[FeedImage]>?
+	var onErrorStateChange: Observer<Bool>?
```

Comment:
Ideally, the presentation layer should decide which string to use, including localization logic. This way, the controller or view doesn't need to contain presentation logic.

So instead of a boolean, it could be an optional string to represent both cases (visible or not).

```suggestion
	var onErrorStateChange: Observer<String?>?
```